### DISCUSSION TITLE
MoreMcMeta cannot be applied to Custom item models?

### DISCUSSION THREAD

**Discussion by V-lpen** [MoreMcmeta/core#75] [Core Dev Questions]
I am trying to apply a glow to CustomModelData models but I cant seem to get it working. Am i doing something wrong?` /give @s clay_ball{CustomModelData:1}` . I have the texture set to do _e but i also tried with a .moremcmeta file and neither would load on the model. 
[moremcmeta test.zip](https://github.com/MoreMcmeta/core/files/15174396/moremcmeta.test.zip)

**Comment by soir20** [comment]
Thanks for providing a sample resource pack. Are you using Forge or Fabric?

**Reply by V-lpen** [reply]
i realized i replied to the wrong thing my bad, dunno if you saw my other comment!!
Im using forge for this! Though I forgot to test on fabric, it was late when I was testing this out. This mod is awesome by the way!

**Reply by soir20** [reply]
Glad you like the mod! Just FYI, I haven't forgotten about this; I've had a lot of work on my plate recently so I am still working through the backlog of issues.

**Reply by Kn0wnG1itch** [reply]
I'm not trying to highjack the discussion, but I was wondering if this is what problem I'm having. So when I use Forge CIT (the forge port of Citresewn), the custom textures apply but the emissive texture layer does not. But when I have the emissive texture setup in a default way (like having the item always have that texture, ignoring any conditions), then the emissive texture works correctly. And it works if I place the CIT textures into the minecraft/textures/item folder and having the textures named something like diamond_sword.png and diamond_sword_e.png,  

(I've confused myself with CIT and emissive textures recently when trying to get them to work, so I just want to make sure that the problems are the same.)
